woocommerce/woocommerce-gutenberg-products-block
# Update placeholder images to reflect new wireframe designs

In #5049 we introduced temporary placeholder SVGs. 
This PR replaces the temporary placeholders with the new designs.

Closes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5053



|Product Archive|Single Product|
|-|-|
|![archive-product-min](https://user-images.githubusercontent.com/1562646/140085376-db5f9bfd-b0a4-401d-846c-3bf16e0f8269.jpg)|![single-product-min](https://user-images.githubusercontent.com/1562646/140085535-2726ba38-59ae-4924-b6aa-7badc4795202.jpg)|


### Manual Testing

How to test the changes in this Pull Request:

**Prerequisite**: Please ensure you have a Block Template Theme activated such as TT1 and also the Gutenberg Plugin installed.

1. Load the Site Editor (Appearance > Editor) and select the `Product Archive Page` template from the `General templates` section.
2. Confirm the placeholder reflects the screenshot above.
3. Repeat for the `Single Product Page` template.
